src/System.Memory/tests/ReadOnlySpan/Contains.char.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    // Adapted from IndexOf.byte.cs
+    public static partial class ReadOnlySpanTests // .Contains<Char>
```

Comment:
Similar to the previous Contains tests, I would merge this with string tests: https://github.com/dotnet/corefx/pull/29935

i.e. Make sure add a span-based assert here:
https://github.com/dotnet/corefx/blob/a99a57f59f8152099e7da3e2c623568ee9cc016e/src/System.Runtime/tests/System/StringTests.netcoreapp.cs#L109

Reply:
Done